### PR TITLE
mgr/dashboard: replace dashboard with dashboard_v2

### PR DESCRIPTION
**Pull request by rjfd**
This PR removes the `dashboard` module from the source tree, and renames `dashboard_v2` to `dashboard`.

Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: qa/tasks/mgr/test_dashboard.py [hunk 1]
```diff
@@ -48,12 +45,7 @@ def test_urls(self):
         # the content is correct or even renders!
 
         urls = [
-            "/health",
-            "/servers",
-            "/osd/",
-            "/osd/perf/0",
-            "/rbd_mirroring",
-            "/rbd_iscsi"
+            "/",
```
Comment (sebastian-philipp) [inline qa/tasks/mgr/test_dashboard.py]: Actually, we should also test our controllers in the smoke test. But that's a different PR

File: src/pybind/mgr/dashboard/module.py [hunk 2]
```diff
@@ -1,1287 +0,0 @@
-
-"""
-Demonstrate writing a Ceph web interface inside a mgr module.
```
Comment (jcsp) [inline src/pybind/mgr/dashboard/module.py]: Goodbye dashboard v1, you did your job!

File: qa/tasks/mgr/test_dashboard.py (hunk 1 above)
Comment (jecluis) [reply qa/tasks/mgr/test_dashboard.py]: Is this something you feel should be addressed in this PR?

**Comment by rjfd** [comment]
http://pulpito.ceph.com/rdias-2018-03-15_22:11:15-rados-wip-rdias-testing-distro-basic-smithi/

Only two failures, which I think are unrelated with this PR.

**Pull request merged**